crates/fuel-core/src/service/adapters/graphql_api.rs

```diff
@@ -130,6 +133,13 @@ impl BlockProducerPort for BlockProducerAdapter {
             .dry_run(transactions, height, time, utxo_validation, gas_price)
             .await
     }
+
+    async fn storage_read_replay(
+        &self,
+        height: BlockHeight,
+    ) -> anyhow::Result<Vec<StorageReadReplayEvent>> {
+        self.block_producer.storage_read_replay(height).await
```

Comment:
I think we need to add a back pressure here. Or use `tokio_rayon::spawn_fifo` like we do for `dry_run` inside of the `BlockProducer`

Reply:
Right. Done in aaaa57bba6.